db/schema.rb

```diff
@@ -659,6 +659,15 @@
   add_index "organisation_visits", ["organisation_id"], name: "index_organisation_visits_on_organisation_id", using: :btree
   add_index "organisation_visits", ["visit_id", "organisation_id"], name: "index_organisation_visits_on_visit_id_and_organisation_id", unique: true, using: :btree
 
+  create_table "specifics", force: :cascade do |t|
```

Comment:
I'm kind of wondering if specifics is necessary. It seems like something that people will try to use at first, then run into trouble because it's too limited. Also I fear that it's a source of pain when many plugins are using them and they clash/overwrite each other.

I wonder about not including specifics and just encouraging plugin authors to write their own models from the get go